sag_et.par

```diff
@@ -15,88 +19,127 @@ n 14 8
 n 15 9
 /|  N\  /N\
 n 16 9
-)N(  M)
+|)  (|\
 d 17 10
 /|\  /N\
 d 19 11
 /N\
+n 20 12
+)~|  (|\  ~N\  /N\
 n 21 12
-|)  |\
+|)  (|\
 d 22 13
 s 11
 /|  N\  /N\
 n 23 13
-)N(  M)  (X(
+|)  |\  (|\
 d 24 14
 s 8
 /|\  /N\
+n 25 15
+)~|  /|  N\  ~N\  /N\
 d 26 15
+s 13
 /N\
 d 27 16
-/|  /|)  N\  /N\
+/|  (|\  N\  /N\
+n 28 16
+|)  |\  (|\
 d 29 17
 /|  N\  /N\
+n 30 18
+)|  /|  (|\  N\  (N~  /N\
 d 31 18
 /|\  /N\
+n 32 19
+)~|  /|  N\  ~N\  /N\
+n 33 19
+|)  |\  (|\
 d 34 20
 /|  /|\  N\  /N\
+n 35 20
+|(  |)  (|\
 d 36 21
-s 9 18

```

Comment:
I did not intend to remove the notation for 9EDO as a subset of 36, however, unless I did this, Scala would throw an error:

`sag_et.par: no symbol for n 9 5`

It remains unclear why removing the subset notation resolves the error. Without visibility into Scala's implementation of Sagittal, I only had my observations from experiments to work with. I tried many different things but this was the only change I could get to work without error.

Reply:
Beats me too. But I think it's OK to delete the notation for 9 as a subset of 36, for now. But we should raise it with Manuel.

Reply:
Noted.